CHANGELOG.md

```diff
@@ -55,6 +55,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
+* Fix `test_spec` target generation to pass `pod lib lint`  
```

Comment:
@paulz this changelog entry still does not make sense or convey enough information on what the change actually was.

I would split this into two commits and two changelog entries that includes:

1) Create a generic Info.plist file for test targets
2) Use `PRODUCT_MODULE_NAME` for generated test targets

Reply:
will do